eng/versioning/version_client.txt

```diff
@@ -44,6 +44,8 @@ com.azure:perf-test-core;1.0.0-beta.1;1.0.0-beta.1
 # note: The unreleased dependencies will not be manipulated with the automatic PR creation code.
 unreleased_com.azure:azure-core;1.4.0-beta.1
 unreleased_com.azure:azure-core-amqp;1.1.0-beta.1
+unreleased_com.azure:azure-core-test;1.2.0-beta.1
+unreleased_com.azure:azure-core;1.4.0-beta.1
```

Comment:
@sima-zhu this is the reason you're seeing the analyze step fail. There are two unreleased_com.azure:azure-core;1.4.0-beta.1 entries, one on line 48 and the other on line 43. When a duplicate entry is encountered my script was doing a write-error which apparently caused the wrapper script to exit without flushing the buffer first so it was never seen. I'll be fixing my script so that doesn't happen again.